<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>359</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LOGIN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(134, 189, 255);
border-color: rgb(13, 13, 13);</string>
  </property>
  <widget class="QLabel" name="lUsuario">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>USUARIO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lClave">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CONTRASEÃ‘A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lAviso">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>335</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>juan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lIcono">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recursos.qrc">:/imagenes/imagenes/auto.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Boton" name="widgetIngresar" native="true">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Boton</class>
   <extends>QWidget</extends>
   <header>boton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
